### PR TITLE
Add AdRotator unit tests

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

**Comment by csharpfritz** [comment]
@hishamco can you fix these test failures before we merge this PR?

**Comment by hishamco** [comment]
There's something strange about this PR, it passed when I debug, but it fails without ;)

let me check again ...

File: src/BlazorWebFormsComponents.Test/TestComponent.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using System.IO;
+using Bunit;
+
+namespace BlazorWebFormsComponents.Test
+{
+  public class TestComponent : TestComponentBase
+  {
+		public TestComponent()
+		{
+			ContentRoot = new DirectoryInfo(Environment.CurrentDirectory).Parent.Parent.Parent.FullName;
```
[the remark this answers is not in the document]
Comment (hishamco) [reply src/BlazorWebFormsComponents.Test/TestComponent.cs:11]: You are right

[the remark this answers is not in the document]
Comment (hishamco) [reply src/BlazorWebFormsComponents.Test/TestComponent.cs:11]: Seems I already did that, but I forgot to commit the changes ;)

**Comment by hishamco** [comment]
Looks great now, thanks @grleachman

@csharpfritz please :shipit:

**Pull request merged**